examples/hitl/basic_viewer/README.md

```diff
@@ -9,11 +11,8 @@ See [siro_sandbox/README.md](../../siro_sandbox/README.md).
 
 ```bash
 HABITAT_SIM_LOG=warning MAGNUM_LOG=warning \
-python examples/hitl/basic_viewer/basic_viewer.py \
---disable-inverse-kinematics \
---never-end \
---cfg examples/hitl/basic_viewer/config/basic_viewer.yaml
+python examples/hitl/basic_viewer/basic_viewer.py habitat_hitl.episodes_filter='0 2 4 10:15 1000:4000:500'
```

Comment:
This app crashes on my setup with the following error:

```
ImageView: data too small, got 170640 but expected at least 170880 bytes
``` 

I'll investigate further.

Reply:
This is this issue: https://app.asana.com/0/1205829436198608/1206291063244943

Reply:
I guess I can remove the debug image usage from basic_viewer for now, and we'll make an Asana item to reintroduce them once the ImageViewer bug is fixed.

Reply:
Fixed in #1754 . https://github.com/facebookresearch/habitat-lab/pull/1754/commits/2a4bb832f78d8d22d894029fd03fb481a2f23416 .